docs/azure-saas-docs/content/en/quick-start.md

```diff
@@ -0,0 +1,76 @@
+
+After finishing the IDP setup, you may choose to either run the project locally first or immediately deploy the solution to Azure.
+## 2.a. Running the Dev Kit in your local dev environment
+
+- Install the latest version of [Visual Studio 2022](https://visualstudio.microsoft.com/vs/). You may also use Visual Studio Code, but the solution and projects are targetted at VS2022.
+- Clone the repository `https://github.com/Azure/azure-saas.git` on to your dev machine.
+- Open the `.sln` in the root of the repository. This solution includes all of the modules.
+- Depending on the project you wish to run, you'll need to set some secrets to properly setup authentication with B2C. See the [App Settings](#app-settings) section below
+
+### App Settings
+When running the project locally, you will need to set some app settings & secrets manually using the [.NET secrets manager](https://docs.microsoft.com/en-us/aspnet/core/security/key-vault-configuration?view=aspnetcore-6.0#secret-storage-in-the-development-environment). When deployed to azure (below), these secrets are automatically configured for you and stored in Azure Key Vault.
+
+Make sure you check out the [readme files](#more-info) in each project's directory for a description of the app settings & secrets you'll need to set in order to run the respective project.
+
+
+## 2.b. Deploying to Azure - Entire Solution
+
+Deploying to Azure is easy thanks to our pre-configured ARM (Azure Resource Manager) templates.
+
+This button will take you to the Azure portal, passing it the template. You'll be asked a few questions, and then the solution will be up and running in just a few minutes. You will need your B2C configuration values and secrets from step 1.
+
+**Deployment Coming Soon!**
+<!-- [![Deploy to Azure](https://www.azuresaas.net/assets/images/deploy-to-azure.svg)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-saas%2Fmain%2Fsrc%2FSaas.Deployment%2FSaas.Deployment.Root%2Fazuredeploy.json/createUIDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-saas%2Fmain%2Fsrc%2FSaas.Deployment%2FSaas.Deployment.Root%2FcreateUiDefinition.json) -->
+
+## 2.c. (Advanced) Deploying to Azure - Single Module
+
+If you'd like to use just one (or more) module from the project, we've provided bicep templates to do that as well. In each project directory, you'll find a folder named `{ModuleName}.Deployment` that contains all the bicep code you'll need to deploy just that Module. Please be advised that there are certain dependencies that each module requires in order for it to deploy properly. You may find that you need to edit the bicep templates to match your use case. You will find instructions and a list of dependencies for each module within the [module's readme](#more-info).
```

Comment:
Since this section indicates that modifying the Bicep templates may be necessary for individual deployment, it it probably a good idea to link to the Bicep documentation here.